Lib/datetime.py

```diff
@@ -1033,7 +1033,11 @@ def ctime(self):
             self._day, self._year)
 
     def strftime(self, fmt):
-        "Format using strftime()."
+        """
+        Format using strftime().
+        
+        Example: "%d/%m/%Y, %H:%M:%S"
+        """
```

Comment:
```suggestion
        """
        Format using strftime().

        Example: "%d/%m/%Y, %H:%M:%S"
        """
```

We require to remove meaningless spaces.